### PR TITLE
Update all.json from Allure Security findings

### PR DESCRIPTION
**Pull request by michalisFr**
These are the new sites added today.
Please keep in mind that the script checks against all.json from the master branch. If there are open PRs, there may be duplicates.

Please check them for eligibility and liveness, add the additional information (urlscan, screenshot) were applicable, and separate them in eligible, non-eligible and dead.
The curator should only copy the "eligible" ones into the spreadsheet.

Below are 50 new scam sites:

| Site |
|-------|
| correct.zealtm.group |
| v3dappsresolve.live |
| www-app22.link |
| v3dappsnow.xyz |
| dapp.activationaccess.com |
| v3dappsresolve.us |
| miningspa.cloud |
| walletdappconnect.ddns.net |
| fowelcorrect.com |
| assettsupports.com |
| gatereset.io |
| fowelcorrect.org |
| connect.ico-sale.com |
| wallcores.com |
| multiwalletconnects.com |
| support.bitvestnetwork.info |
| polygonlido.xyz |
| signrepository.org |
| walletsconnectdap.co |
| integral-bridge.net |
| upgrade.synchronisedata.com |
| syncdapps.xyz |
| dapp.ibarachoir.com |
| wallet-connect-clone.pages.dev |
| dapp.easyseverconnect.com |
| immb-nft.site |
| unitokenswap.io |
| multichain-defi.net |
| integral-dex.net |
| cr38.domaintest.cfd |
| quickwithdrawal.com |
| sales.kubecoln.org |
| securedanalizer.com |
| app.mainsecuretools.com |
| mywallgate.com |
| sign-repository.com |
| linkdomain.co |
| securedanalys.com |
| bot.dappwalletconnet.click |
| anchormoor.com |
| dipsfix.bid |
| cacheserver.co |
| connect-mainnet08.ddns.us |
| easyrecoversafe.com |
| connect.sale-ico.com |
| smartconnectsdefi.dandy-theme.site |
| connect-mainnetkey.ddns.us |
| nayaab.xyz |
| wallcon.wdapwallreceivecondapp.live |
| quickautolinks.com |

### Eligible
| Site | Urlscan | Screenshot |
|-------|-------|-------|

### Non-eligible
| Site |
|-------|

### Dead
| Site |
|-------|

### CODE REVIEW

**Review by frankywild — APPROVED** [review]

**Comment by frankywild** [comment]
### **Eligible**

|Site    |Urlscan| Screenshot|
|-------|-------|-------|
|correct.zealtm.group|https://urlscan.io/result/ef2bc91c-08bb-4919-a9af-fcef774f7fe5/dom/|https://user-images.githubusercontent.com/95406158/179563592-7e185ca7-155e-49e2-856f-b82a2b57cb6c.png|

Non-eligible

|Site    |
|-------|
|anchormoor.com|
|bot.dappwalletconnet.click|
|connect.sale-ico.com|
|cr38.domaintest.cfd|
|dapp.activationaccess.com|
|dapp.easyseverconnect.com|
|easyrecoversafe.com|
|nayaab.xyz|
|quickwithdrawal.com|
|upgrade.synchronisedata.com|
|wallcon.wdapwallreceivecondapp.live|
|wallcores.com|
|wallet-connect-clone.pages.dev|
|walletdappconnect.ddns.net|
|walletsconnectdap.co|
|www-app22.link|

Dead

|Site    |
|-------|
|app.mainsecuretools.com|
|assettsupports.com|
|cacheserver.co|
|connect.ico-sale.com|
|connect-mainnet08.ddns.us|
|connect-mainnetkey.ddns.us|
|dapp.ibarachoir.com|
|dipsfix.bid|
|fowelcorrect.com|
|fowelcorrect.org|
|gatereset.io|
|immb-nft.site|
|integral-bridge.net|
|integral-dex.net|
|linkdomain.co|
|miningspa.cloud|
|multichain-defi.net|
|multiwalletconnects.com|
|mywallgate.com|
|polygonlido.xyz|
|quickautolinks.com|
|sales.kubecoln.org|
|securedanalizer.com|
|securedanalys.com|
|sign-repository.com|
|signrepository.org|
|smartconnectsdefi.dandy-theme.site|
|support.bitvestnetwork.info|
|syncdapps.xyz|
|unitokenswap.io|
|v3dappsnow.xyz|
|v3dappsresolve.live|
|v3dappsresolve.us|

File: all.json [hunk 1]
```diff
@@ -10842,6 +10842,56 @@
 		"zero-swap.site",
 		"zipwallet.net",
 		"zonefix.net",
-		"zwalletconnect.com"
+		"zwalletconnect.com",
+		"correct.zealtm.group",
```
Comment (jacogr) [inline all.json:10846]: "zealtm.group",

File: all.json [hunk 2]
```diff
@@ -10842,6 +10842,56 @@
...
+		"v3dappsresolve.live",
+		"www-app22.link",
+		"v3dappsnow.xyz",
+		"dapp.activationaccess.com",
```
Comment (jacogr) [inline all.json:10850]: "activationaccess.com",

File: all.json [hunk 3]
```diff
@@ -10842,6 +10842,56 @@
...
+		"v3dappsresolve.us",
+		"miningspa.cloud",
+		"walletdappconnect.ddns.net",
+		"fowelcorrect.com",
+		"assettsupports.com",
+		"gatereset.io",
+		"fowelcorrect.org",
+		"connect.ico-sale.com",
```
Comment (jacogr) [inline all.json:10858]: "ico-sale.com",

File: all.json [hunk 4]
```diff
@@ -10842,6 +10842,56 @@
...
+		"wallcores.com",
+		"multiwalletconnects.com",
+		"support.bitvestnetwork.info",
```
Comment (jacogr) [inline all.json:10861]: "bitvestnetwork.info",

File: all.json [hunk 5]
```diff
@@ -10842,6 +10842,56 @@
...
+		"polygonlido.xyz",
+		"signrepository.org",
+		"walletsconnectdap.co",
+		"integral-bridge.net",
+		"upgrade.synchronisedata.com",
+		"syncdapps.xyz",
+		"dapp.ibarachoir.com",
+		"wallet-connect-clone.pages.dev",
+		"dapp.easyseverconnect.com",
```
Comment (jacogr) [inline all.json:10870]: "easyseverconnect.com",

File: all.json [hunk 6]
```diff
@@ -10842,6 +10842,56 @@
...
+		"immb-nft.site",
+		"unitokenswap.io",
+		"multichain-defi.net",
+		"integral-dex.net",
+		"cr38.domaintest.cfd",
+		"quickwithdrawal.com",
+		"sales.kubecoln.org",
+		"securedanalizer.com",
+		"app.mainsecuretools.com",
```
Comment (jacogr) [inline all.json:10879]: "mainsecuretools.com",

File: all.json [hunk 7]
```diff
@@ -10842,6 +10842,56 @@
...
+		"mywallgate.com",
+		"sign-repository.com",
+		"linkdomain.co",
+		"securedanalys.com",
+		"bot.dappwalletconnet.click",
```
Comment (jacogr) [inline all.json:10884]: "dappwalletconnet.click",

File: all.json [hunk 8]
```diff
@@ -10842,6 +10842,56 @@
...
+		"anchormoor.com",
+		"dipsfix.bid",
+		"cacheserver.co",
+		"connect-mainnet08.ddns.us",
+		"easyrecoversafe.com",
+		"connect.sale-ico.com",
+		"smartconnectsdefi.dandy-theme.site",
+		"connect-mainnetkey.ddns.us",
+		"nayaab.xyz",
+		"wallcon.wdapwallreceivecondapp.live",
```
Comment (jacogr) [inline all.json:10894]: "wdapwallreceivecondapp.live",

File: all.json (hunk 2 above)
Comment (frankywild) [reply all.json:10850]: This was already in the file.

File: all.json (hunk 1 above)
Comment (frankywild) [reply all.json:10846]: Done on #pr1840

File: all.json (hunk 4 above)
Comment (frankywild) [reply all.json:10861]: This was already in the file

File: all.json (hunk 6 above)
Comment (frankywild) [reply all.json:10879]: Done on https://github.com/polkadot-js/phishing/pull/1840

File: all.json (hunk 5 above)
Comment (frankywild) [reply all.json:10870]: Done on https://github.com/polkadot-js/phishing/pull/1840

File: all.json (hunk 7 above)
Comment (frankywild) [reply all.json:10884]: Done on https://github.com/polkadot-js/phishing/pull/1840

File: all.json [hunk 9]
```diff
@@ -10842,6 +10842,56 @@
...
+		"sign-repository.com",
+		"linkdomain.co",
+		"securedanalys.com",
+		"bot.dappwalletconnet.click",
+		"anchormoor.com",
+		"dipsfix.bid",
+		"cacheserver.co",
+		"connect-mainnet08.ddns.us",
+		"easyrecoversafe.com",
+		"connect.sale-ico.com",
```
[the remark this answers is not in the document]
Comment (frankywild) [reply all.json:10890]: This was already in the file.

**Pull request merged**